telemetry/core/src/main/scala/io/buoyant/telemetry/default.scala

```diff
@@ -0,0 +1,30 @@
+package io.buoyant.telemetry
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.twitter.finagle.Stack
+import com.twitter.finagle.stats.{DefaultStatsReceiver, NullStatsReceiver}
+import com.twitter.finagle.tracing.{DefaultTracer, NullTracer}
+
+class DefaultInitializer extends TelemeterInitializer {
```

Comment:
Similar comment here as in #594 -- for consistency with other initializers, I'd capitalize the filename.  Would also consider calling it DefaultTelemetryInitializer.scala, similar to DefaultInterpreterInitializer.scala.